modules/learn/pages/services-and-indexes/services/services.adoc

```diff
@@ -39,7 +39,9 @@ workload-requirements.
 
 Services are set up on a per node basis.
 Each node can run at most one instance of a service.
-The _Data Service_ must run on at least one node.
+Each node can run any number of services, up to the maximum, which is seven.
+In Couchbase Enterprise Server Version 7.6+, a node can be configured to run _no_ service.
+The _Data Service_ must run on at least one node of the cluster.
```

Comment:
I'm not sure this is important here but....The first node of the cluster (the one that creates the cluster) must have the data service configured.